mailbox/backup/pom.xml

```diff
@@ -43,9 +44,46 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>apache-james-mailbox-memory</artifactId>
```

Comment:
This dependency should not be needed out of `test`

Reply:
You're totally right.
it's from JAMES-2681, i think i have fixed it in it. but not rebased yet in this one

Reply:
Just fixed in JAMES-2681 :)